### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by AnniqueN**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -40,3 +40,61 @@ const siteContent = {
...
+nav.prepend(newLastNav);
+let navItems = document.querySelectorAll("a");
+navItems[0].textContent = siteContent.nav["nav-item-1"];
+navItems[1].textContent = siteContent.nav["nav-item-2"];
+navItems[2].textContent = siteContent.nav["nav-item-3"];
+navItems[3].textContent = siteContent.nav["nav-item-4"];
+navItems[4].textContent = siteContent.nav["nav-item-5"];
+navItems[5].textContent = siteContent.nav["nav-item-6"];
+
+navItems.forEach( (cv) => {return cv.style.color = "green"});
```
Comment (Mykolb) [inline js/index.js:59]: Smart!

File: js/index.js [hunk 2]
```diff
@@ -40,3 +40,61 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let newFirstNav = document.createElement("a");
```
Comment (Mykolb) [inline js/index.js:44]: Everything looks good! Nice job adding the nav elements

**Comment by Mykolb** [comment]
MVP met! Great job!

I need you to add me as a collaborator so that I can close the pull request please and thank you!

**Comment by AnniqueN** [comment]
> MVP met! Great job!
> 
> I need you to add me as a collaborator so that I can close the pull request please and thank you!

I thought I did, sorry about that... yesterday was rough...

**Comment by Mykolb** [comment]
Thank you and I hope you feel better!

**Comment by AnniqueN** [comment]
> Thank you and I hope you feel better!

I am much better... thank you!

**Pull request merged**